### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by 1234hdpa**
Add some new translation lines.

### CODE REVIEW

File: Translations/vi.lproj/FinderExtensionLocalizable.strings [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-"Extract to..." = "Extract to...";
-"Extract to \"%@\"" = "Extract to \"%@\"";
-"Compress with Keka" = "Compress with Keka";
+"Open with Keka" = "M? v?i Keka";
+"Compress here" = "N?n t?i ??y";
+"Compress to..." = "N?n ??n...";
+"Extract here" = "Gi?i n?n t?i ??y";
+"Extract to..." = "Gi?i n?n ??n...";
+"Extract to \"%@\"" = "Gi?i n?n ??n \"%@\"";
+"Compress with Keka" = "N?n v?i Keka";
```
Comment (aonez) [inline Translations/vi.lproj/FinderExtensionLocalizable.strings:7]: Are those rendered bad?

**Review by aonez — CHANGES_REQUESTED** [review]
Thanks a lot @1234hdpa! I think some strings have broken characters. Can you check?

File: Translations/vi.lproj/FinderExtensionLocalizable.strings (hunk 1 above)
Comment (1234hdpa) [reply Translations/vi.lproj/FinderExtensionLocalizable.strings:7]: OK. I'll check it back. Thanks.

Comment (1234hdpa) [reply Translations/vi.lproj/FinderExtensionLocalizable.strings]: I'd just replaced these broken words via Xcode. Visual Studio Code doesn't work well as saving these.

**Review by aonez — APPROVED** [review]

File: Translations/vi.lproj/FinderExtensionLocalizable.strings (hunk 1 above)
Comment (aonez) [reply Translations/vi.lproj/FinderExtensionLocalizable.strings]: Never used Visual Studio in Mac but probably the encoding for that file was not properly set. Thanks for the fix!

**Comment by aonez** [comment]
Web updated already 😊 maybe you want to translate this line too:

https://github.com/aonez/Keka/blob/c840b72b94ff13fa989e585281fd62cfc7067e46/Translations/Web/lang_vi.php#L29

**Comment by aonez** [comment]
@1234hdpa just realized the Vietnamese translation is not found in the credits! Just went ahead and added you. Change your name/link if you want 😊

https://github.com/aonez/Keka/blob/941036f9dbdb404ff7ab77527d40373704f923da/Translations/Common/Translators.html#L160-L164

**Comment by 1234hdpa** [comment]
Thanks for adding me to the credits!
I'd just added some words for "Helper" and made it easy to understand.

**Comment by aonez** [comment]
> I'd just added some words for "Helper" and made it easy to understand.

Where? You'll create a new pull request?

**Comment by 1234hdpa** [comment]
My bad 😅! I forgot it.

**Pull request merged**